python/emitter.js

```diff
@@ -128,10 +116,35 @@
     addHook('beforeEach', python`    return conf`)
     addHook('beforeEach', python`\n`)
 
-    if (mobile) setUpMobileNative(test, addHook)
-    else {
-        if (emulator) setUpWithEmulators(test, addHook)
-        else setUpBrowsers(test, addHook)
+    if (mobile) {
+        if(test.env.app) {
+            addHook('beforeEach', python`@pytest.fixture(scope="function")`)
+            addHook('beforeEach', python`def app():`)
+            addHook('beforeEach', python`    return ${test.env.app}\n`)
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=8942416)
```suggestion
            addHook("beforeEach", python`    return ${test.env.app}\n`)
```